<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMail adrese</string>
  </property>
  <property name="windowIcon">
   <iconset resource="newwindow.qrc">
    <normaloff>:/Ikone/window_logo.png</normaloff>:/Ikone/window_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="iipLabel">
        <property name="text">
         <string>Ime i prezime:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="iipLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="napomenaLabel">
        <property name="text">
         <string>Napomena:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="napomenaLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="savePushButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearAllPushButton">
        <property name="text">
         <string>Clear all</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionDeleteRecord"/>
   <addaction name="actionDeleteAll"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/save.png</normaloff>:/Ikone/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/new.png</normaloff>:/Ikone/new.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/close.png</normaloff>:/Ikone/close.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/open.png</normaloff>:/Ikone/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDeleteRecord">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/delete.png</normaloff>:/Ikone/delete.png</iconset>
   </property>
   <property name="text">
    <string>DeleteRecord</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="icon">
    <iconset resource="newwindow.qrc">
     <normaloff>:/Ikone/clear.png</normaloff>:/Ikone/clear.png</iconset>
   </property>
   <property name="text">
    <string>DeleteAll</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="newwindow.qrc"/>
 </resources>
 <connections/>
</ui>
